ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,21 @@
+ages = [29 , 19 , 17 , 26 , 32 , 16 , 21]
+print(ages)
+print(ages[4])
+print(ages[-2])
+print(ages[2:6])
+print(ages[:4])
+print("17" in ages)
+print("42" in ages)
```

Comment:
Check for the values `17` and `42`, not the values `"17"` and `"42"`. Strings are different from integers.

---

![image](https://user-images.githubusercontent.com/16143771/149631927-a19d6573-0aa5-4f96-b934-3517988caa75.png)

---

Since `ages` _does_ contain the value `17`, its output should be `True`, not `False`.

---

![image](https://user-images.githubusercontent.com/16143771/149632159-6a67401f-6d4f-4e64-a34b-317a1ad8b276.png)

---

![image](https://user-images.githubusercontent.com/16143771/149632163-ab2b1283-156d-494e-a09f-5fbdf36091e8.png)